hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,119 @@
+ms.topic: article
+ms.date: 3/14/2023
+keywords: uno platform, winui
+ms.author: alvinashcraft
+author: lukeblevins
+ms.localizationpriority: medium
+ms.custom: template-quickstart
+audience: new-desktop-app-developers
+content-type: how-to
+---
+
+# How to target multiple platforms with your WinUI 3 app
+
+Now that you've created your first WinUI 3 app, you might be wondering how to reach more users with a single codebase. This how-to will use Uno Platform to expand the reach of your existing application enabling reuse of the business logic and UI layer across native mobile, web, and desktop.
+
+:::image type="content" source="images/hello-world/uno-hello-world-web.png" alt-text="The 'Hello world' app running in the browser.":::
+
+## Prerequisites
+
+- [Visual Studio 2022 17.4 or later](https://visualstudio.microsoft.com/#vs-section)
+- [Tools for Windows App SDK](../windows-app-sdk/set-up-your-development-environment.md)
+- ASP.NET and web development workload (for WebAssembly development)
+- .NET Multi-platform App UI development installed (for iOS, Android, Mac Catalyst development).
+- .NET desktop development installed (for Gtk, Wpf, and Linux Framebuffer development)
+
+## Finalize your environment
+
+1. Open a command-line prompt, Windows Terminal if you have it installed, or else Command Prompt or Windows Powershell from the Start Menu.
+
+2. Install the `uno-check` tool:
+    - Use the following command:
+
+        `dotnet tool install -g uno.check`
+
+    - To update the tool, if you already have an existing one:
+
+        `dotnet tool update -g uno.check`
+
+3. Run the tool with the following command:
+
+    `uno-check`
+
+4. Follow the instructions indicated by the tool.
+
+## Install the solution templates
+
+1. Launch Visual Studio 2022, then click `Continue without code`. Click `Extensions` -> `Manage Extensions` from the Menu Bar.
+
+    :::image type="content" source="images/hello-world/manage-extensions.png" alt-text="Visual Studio Menu bar item that reads manage extensions":::
+
+2. In the Extension Manager expand the **Online** node and search for `Uno`, install the `Uno Platform` extension or download it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022), then restart Visual Studio.
+
+    :::image type="content" source="images/hello-world/uno-extensions.PNG" alt-text="Visual Studio Menu bar item that reads manage extensions":::
+
+## Create an application
+
+Now that we are ready to create a multi-platform application, the approach we'll take is to create a new Uno Platform application, and then copy the existing WinUI 3 project items into it. This will allow you to reuse the existing codebase, and then you can add the necessary changes over time to make new functionality work on other platforms. This approach is also useful if you have an existing application that you want to port to other platforms.
```

Comment:
I recommend naming the 'existing' project , presumably "Hello World" WinUI sample from previous tutorial